Tweak report colors so that we are WCAG2AA valid.

XhmikosR opened the pull request:
| Before | After |
| ------------- |-------------| 
| ![1before](https://cloud.githubusercontent.com/assets/349621/23793455/03223c62-0594-11e7-9aa6-35f835f6e1d7.png) | ![2after](https://cloud.githubusercontent.com/assets/349621/23821804/5b23e6ec-0646-11e7-8b7b-bda3f7f01ce4.png)  |

Feel free to tweak this if you want to :)

patrickhulce commented:
I'm surprised this is more valid than the previous, I can barely distinguish warning/error/poor score after this change! This has seems like a tug of war with the brown color, and a trade-off between background contrast and contrast between items. IMO, we shouldn't change as moving from AA to AAA weakens the latter significantly.

XhmikosR commented:
That is why I say feel free to tweak this :)

It's clear though that the current colors do **not** pass WCAG 2AA on certain occasions.

For example the average bacjground color is `#ef6c00` on `#fff` which gives a 3.1 contrast ration for large text only. So I decided to go with WCAG2AAA, but I can revert some of the changes so that at least it's WCAG2AA valid.

patrickhulce commented:
Instead of weakening all of the colors further, how about we flip the score text to black? That yields good contrast ratios (5+) and still keeps inter-item comparisons scannable.

XhmikosR commented:
I thought about that too. I'll give it a go later unless one of you beats
me to it :)

On Mar 10, 2017 19:49, "Patrick Hulce" <notifications@github.com> wrote:

> Instead of weakening all of the colors further, how about we flip the
> score text to black? That yields good contrast ratios (5+) and still keeps
> inter-item comparisons scannable.
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/GoogleChrome/lighthouse/pull/1851#issuecomment-285736928>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/AAVVtVfvHPMdPjWlMT2mIGtQmjPEcyDJks5rkY0SgaJpZM4MZRqr>
> .
>

patrickhulce commented:
Sweet, appreciate the vigilance on these and contributions @XhmikosR ! :)

XhmikosR commented:
@patrickhulce: now everything is passing for me for WCAG2AA and for WCAG2AAA there are only 12 issues with this branch; before it was like 70 or so.

ebidel commented:
Hmm. Yea the new colors were chosen to be AA compliant. IMO they look good
as is and we shouldn't continue to tweak them every couple of weeks.

On Sat, Mar 11, 2017, 5:30 AM XhmikosR <notifications@github.com> wrote:

> @patrickhulce <https://github.com/patrickhulce>: now everything is
> passing for me for WCAG2AA and for WCAG2AAA there are only 12 issues with
> this branch; before it was like 70 or so.
>
> —
> You are receiving this because you are subscribed to this thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/GoogleChrome/lighthouse/pull/1851#issuecomment-285747550>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/AAOigDuTdsUbJK7BJTGE2lwsnqVsG8SHks5rkZbegaJpZM4MZRqr>
> .
>

XhmikosR commented:
True but they weren't
AA compliant.

On Mar 11, 2017 00:34, "Eric Bidelman" <notifications@github.com> wrote:

> Hmm. Yea the new colors were chosen to be AA compliant. IMO they look good
> as is and we shouldn't continue to tweak them every couple of weeks.
>
> On Sat, Mar 11, 2017, 5:30 AM XhmikosR <notifications@github.com> wrote:
>
> > @patrickhulce <https://github.com/patrickhulce>: now everything is
> > passing for me for WCAG2AA and for WCAG2AAA there are only 12 issues with
> > this branch; before it was like 70 or so.
> >
> > —
> > You are receiving this because you are subscribed to this thread.
> > Reply to this email directly, view it on GitHub
> > <https://github.com/GoogleChrome/lighthouse/pull/
> 1851#issuecomment-285747550>,
> > or mute the thread
> > <https://github.com/notifications/unsubscribe-auth/
> AAOigDuTdsUbJK7BJTGE2lwsnqVsG8SHks5rkZbegaJpZM4MZRqr>
> > .
> >
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly, view it on GitHub
> <https://github.com/GoogleChrome/lighthouse/pull/1851#issuecomment-285803667>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/AAVVtSbIux6ZfNJ-Anl1MiLQKLdVfjHuks5rkc_ggaJpZM4MZRqr>
> .
>

ebidel commented:
Can you point out where it's failing? When they were updated before they
checked out.

On Sat, Mar 11, 2017, 9:35 AM XhmikosR <notifications@github.com> wrote:

> True but they weren't
> AA compliant.
>
> On Mar 11, 2017 00:34, "Eric Bidelman" <notifications@github.com> wrote:
>
> > Hmm. Yea the new colors were chosen to be AA compliant. IMO they look
> good
> > as is and we shouldn't continue to tweak them every couple of weeks.
> >
> > On Sat, Mar 11, 2017, 5:30 AM XhmikosR <notifications@github.com> wrote:
> >
> > > @patrickhulce <https://github.com/patrickhulce>: now everything is
> > > passing for me for WCAG2AA and for WCAG2AAA there are only 12 issues
> with
> > > this branch; before it was like 70 or so.
> > >
> > > —
> > > You are receiving this because you are subscribed to this thread.
> > > Reply to this email directly, view it on GitHub
> > > <https://github.com/GoogleChrome/lighthouse/pull/
> > 1851#issuecomment-285747550>,
> > > or mute the thread
> > > <https://github.com/notifications/unsubscribe-auth/
> > AAOigDuTdsUbJK7BJTGE2lwsnqVsG8SHks5rkZbegaJpZM4MZRqr>
> > > .
> > >
> >
> > —
> > You are receiving this because you were mentioned.
> > Reply to this email directly, view it on GitHub
> > <
> https://github.com/GoogleChrome/lighthouse/pull/1851#issuecomment-285803667
> >,
> > or mute the thread
> > <
> https://github.com/notifications/unsubscribe-auth/AAVVtSbIux6ZfNJ-Anl1MiLQKLdVfjHuks5rkc_ggaJpZM4MZRqr
> >
> > .
> >
>
> —
> You are receiving this because you commented.
> Reply to this email directly, view it on GitHub
> <https://github.com/GoogleChrome/lighthouse/pull/1851#issuecomment-285803993>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/AAOigGOiYAkgOluHNWxAnGyVT7gzEQYPks5rkdBAgaJpZM4MZRqr>
> .
>

XhmikosR commented:
See my comment above. They are not AS valid and IIRC I mentioned in the
pull request that changed the colors.

On Mar 11, 2017 00:37, "Eric Bidelman" <notifications@github.com> wrote:

> Can you point out where it's failing? When they were updated before they
> checked out.
>
> On Sat, Mar 11, 2017, 9:35 AM XhmikosR <notifications@github.com> wrote:
>
> > True but they weren't
> > AA compliant.
> >
> > On Mar 11, 2017 00:34, "Eric Bidelman" <notifications@github.com> wrote:
> >
> > > Hmm. Yea the new colors were chosen to be AA compliant. IMO they look
> > good
> > > as is and we shouldn't continue to tweak them every couple of weeks.
> > >
> > > On Sat, Mar 11, 2017, 5:30 AM XhmikosR <notifications@github.com>
> wrote:
> > >
> > > > @patrickhulce <https://github.com/patrickhulce>: now everything is
> > > > passing for me for WCAG2AA and for WCAG2AAA there are only 12 issues
> > with
> > > > this branch; before it was like 70 or so.
> > > >
> > > > —
> > > > You are receiving this because you are subscribed to this thread.
> > > > Reply to this email directly, view it on GitHub
> > > > <https://github.com/GoogleChrome/lighthouse/pull/
> > > 1851#issuecomment-285747550>,
> > > > or mute the thread
> > > > <https://github.com/notifications/unsubscribe-auth/
> > > AAOigDuTdsUbJK7BJTGE2lwsnqVsG8SHks5rkZbegaJpZM4MZRqr>
> > > > .
> > > >
> > >
> > > —
> > > You are receiving this because you were mentioned.
> > > Reply to this email directly, view it on GitHub
> > > <
> > https://github.com/GoogleChrome/lighthouse/pull/
> 1851#issuecomment-285803667
> > >,
> > > or mute the thread
> > > <
> > https://github.com/notifications/unsubscribe-auth/AAVVtSbIux6ZfNJ-
> Anl1MiLQKLdVfjHuks5rkc_ggaJpZM4MZRqr
> > >
> > > .
> > >
> >
> > —
> > You are receiving this because you commented.
> > Reply to this email directly, view it on GitHub
> > <https://github.com/GoogleChrome/lighthouse/pull/
> 1851#issuecomment-285803993>,
> > or mute the thread
> > <https://github.com/notifications/unsubscribe-auth/
> AAOigGOiYAkgOluHNWxAnGyVT7gzEQYPks5rkdBAgaJpZM4MZRqr>
> > .
> >
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly, view it on GitHub
> <https://github.com/GoogleChrome/lighthouse/pull/1851#issuecomment-285804417>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/AAVVtbGSns3kmhLXs0Vrh-1tgZAHbtr2ks5rkdDDgaJpZM4MZRqr>
> .
>

ebidel commented:
https://github.com/GoogleChrome/lighthouse/pull/1758 were good to go at 18pt. The good color you mentioned in the PR should have also been fine but I'll check it out when I'm not on mobile.

ebidel commented on lighthouse-core/report/partials/cards.css:
```diff
@@ -30,6 +30,7 @@
   justify-content: center;
   align-items: center;
   border-bottom: 1px solid #ebebeb;
+  color: #4f4f4f
```
;

ebidel commented on lighthouse-core/report/styles/report.css:
```diff
@@ -933,9 +933,9 @@ summary {
 .subitem-result__points {
   margin-top: calc((var(--subitem-line-height) - var(--subitem-font-size) - 4px) / 2);
-  background-color: #000;
+  background-color: fff;
```
`#fff`

ebidel commented on lighthouse-core/report/styles/report.css:
```diff
@@ -933,9 +933,9 @@ summary {
+  background-color: fff;
   padding: 2px 4px;
-  color: #fff;
+  color: #000;
```
IMO, this is pretty hard to read. Since the report is going through a bit of a redesign, let's keep it at `#fff` for now. That's more consistent with the rest of the report.

ebidel commented:
I double checked all the existing colors again on https://snook.ca/technical/colour_contrast/colour.html and they continue to pass "WCAG 2 AA Compliant (18pt+)" per the changes in #1758. So think we're good there. 

Update: I see the PR has changed since the screenshots were posted.

XhmikosR commented:
@ebidel: I reverted the black on white change.

Now, about the previous colors; they were valid only for 18pt or 14px and above.

I went back and and changed them for 2AA.

XhmikosR commented:
Screenshots updated with the current changes.

ebidel commented:
talked with @robdodson, and he suggested we could probably get away with keeping the existing colors if we made the icons slightly bigger (19px). Messing around with that a bit, it does do the trick. I can add those changes to this PR later today or start a new one.

robdodson commented:
19px == 14pt bold which is allowed in wcag2aa. I think the icons are chunky
enough to be considered bold. My hope is this solution presents a middle
ground that satisfies everyone.

On Sun, Mar 12, 2017, 9:37 AM Eric Bidelman <notifications@github.com>
wrote:

> talked with @robdodson <https://github.com/robdodson>, and he suggested
> we could probably get away with keeping the existing colors if we made the
> icons slightly bigger (19px). Messing around with that a bit, it does do
> the trick. I can add those changes to this PR later today or start a new
> one.
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly, view it on GitHub
> <https://github.com/GoogleChrome/lighthouse/pull/1851#issuecomment-285905829>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/ABBFDf8DxRsvDvN5efGIvI1Y0_Y70nL5ks5rkyI5gaJpZM4MZRqr>
> .
>

XhmikosR commented:
Sounds good to me. Although the current state of this branch doesn't have  significant changes to the colors and aims for 2AA.

ebidel commented:
Closing in favor of https://github.com/GoogleChrome/lighthouse/pull/1856

XhmikosR commented:
Cool, give me some time to verify all is good with WCAG2AA in your branch.

On Mar 21, 2017 20:21, "Eric Bidelman" <notifications@github.com> wrote:

> Closing in favor of #1856
> <https://github.com/GoogleChrome/lighthouse/pull/1856>
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly, view it on GitHub
> <https://github.com/GoogleChrome/lighthouse/pull/1851#issuecomment-288172813>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/AAVVtR4sd4kuZN3RHg_zINEIqK0Jzsy0ks5roBUogaJpZM4MZRqr>
> .
>
